e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -538,10 +538,9 @@ describe("issue 25378", () => {
     });
     cy.findByRole("listbox").findByText("months").click();
 
-    popover().within(() => {
-      cy.findByLabelText("Starting from…").click();
-      cy.button("Add filter").click();
-    });
+    popover().findByLabelText("Options").click();
+    popover().last().findByText("Starting from…").click();
+    popover().button("Add filter").click();
```

Comment:
Reverts this
https://github.com/metabase/metabase/pull/44279/files#diff-b8ac0b262f21ddaf28d49d07c061d8919f0013c6980022cafd995aff5b43aa77L540